bio/seqkit/wrapper.py

```
@@ -37,13 +37,17 @@
 )
 
 
+if snakemake.params.get("out_bgzip"):
```

Ruff

Undefined name `snakemake`

(F821)